### PR TITLE
Process TrackedReal

### PR DESCRIPTION
**Pull request by xukai92**
Add optional support for TrackedReal

### CODE REVIEW

**Comment by oxinabox** [comment]
Good idea!
Mght as well do `TrackedArray` too. No?

File: test/Optional/test_Tracker.jl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using TensorBoardLogger, Logging
+using Tracker
+using Test
+
+@testset "Optional: PyPlot.jl" begin
```
Comment (oxinabox) [inline test/Optional/test_Tracker.jl]: Testset name is wrong

**Comment by xukai92** [comment]
Both resolved.

**Review by oxinabox — APPROVED** [review]

**Comment by PhilipVinc** [comment]
Thanks a lot for the contribution!

**Pull request merged**